### PR TITLE
[RHPAM-3322] - Update EAP version from 7.3.4 on PAM/DM 7.10.x

### PR DESCRIPTION
**Pull request by spolti**
See: https://issues.redhat.com/browse/RHPAM-3322

Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by msmagnanijr — APPROVED** [review]

File: businesscentral-monitoring/artifact-overrides.yaml [hunk 1]
```diff
@@ -9,5 +9,5 @@ schema_version: 1
 #                      --overrides-file artifact-overrides.yaml
 
 artifacts:
-  - name: "jboss-eap-7.3.3.GA-CR2-patch.zip"
-    md5: "b627b60cd6bff9cfc5e1f09f3d9c69ae"
+  - name: "jboss-eap-7.3.4.GA-CR2-patch.zip   "
```
Comment (desmax74) [inline businesscentral-monitoring/artifact-overrides.yaml:12]: Extra space here

File: businesscentral/artifact-overrides.yaml [hunk 2]
```diff
@@ -9,5 +9,5 @@ schema_version: 1
 #                      --overrides-file artifact-overrides.yaml
 
 artifacts:
-  - name: "jboss-eap-7.3.3.GA-CR2-patch.zip"
-    md5: "b627b60cd6bff9cfc5e1f09f3d9c69ae"
+  - name: "jboss-eap-7.3.4.GA-CR2-patch.zip   "
```
Comment (desmax74) [inline businesscentral/artifact-overrides.yaml:12]: Extra space here

File: controller/artifact-overrides.yaml [hunk 3]
```diff
@@ -9,5 +9,5 @@ schema_version: 1
 #                      --overrides-file artifact-overrides.yaml
 
 artifacts:
-  - name: "jboss-eap-7.3.3.GA-CR2-patch.zip"
-    md5: "b627b60cd6bff9cfc5e1f09f3d9c69ae"
+  - name: "jboss-eap-7.3.4.GA-CR2-patch.zip   "
```
Comment (desmax74) [inline controller/artifact-overrides.yaml:12]: Extra space here

File: dashbuilder/artifact-overrides.yaml [hunk 4]
```diff
@@ -9,5 +9,5 @@ schema_version: 1
 #                      --overrides-file artifact-overrides.yaml
 
 artifacts:
-  - name: "jboss-eap-7.3.3.GA-CR2-patch.zip"
-    md5: "b627b60cd6bff9cfc5e1f09f3d9c69ae"
+  - name: "jboss-eap-7.3.4.GA-CR2-patch.zip   "
```
Comment (desmax74) [inline dashbuilder/artifact-overrides.yaml:12]: Extra space here

File: kieserver/artifact-overrides.yaml [hunk 5]
```diff
@@ -9,5 +9,5 @@ schema_version: 1
 #                      --overrides-file artifact-overrides.yaml
 
 artifacts:
-  - name: "jboss-eap-7.3.3.GA-CR2-patch.zip"
-    md5: "b627b60cd6bff9cfc5e1f09f3d9c69ae"
\ No newline at end of file
+  - name: "jboss-eap-7.3.4.GA-CR2-patch.zip   "
```
Comment (desmax74) [inline kieserver/artifact-overrides.yaml:12]: Extra space here

File: businesscentral-monitoring/artifact-overrides.yaml (hunk 1 above)
Comment (spolti) [reply businesscentral-monitoring/artifact-overrides.yaml:12]: oh, nice catch.

**Comment by spolti** [comment]
@jakubschwan @Kaitou786 can you please review?

**Review by Kaitou786 — APPROVED** [review]
lgtm

**Pull request merged**